src/python/deepgnn/graph_engine/snark/meta_merger.py

```diff
@@ -4,34 +4,35 @@
 """Merge meta data files generated by distributed converters."""
 import os
 from typing import List
+import json
 
 import fsspec
 
 from deepgnn.graph_engine._base import get_fs
 
-# Content of sample meta.txt
-
-# Fields            Description             Action
-# ====================================================
-# v1                binary version          equal
-# 2708              node_count              additive
-# 10556             edge_count              additive
-# 4                 node_type_num           equal
-# 1                 edge_type_num           equal
-# 2                 node_xxx_feature_num    equal
-# 0                 edge_xxx_feature_num    equal
-# 1                 partitions count        equal
-# 0                 Partition id            equal
-# 140.0             node weight 1           additive
-# 500.0             node weight 2           additive
-# 1000.0            node weight 3           additive
-# 1068.0            node weight 4           additive
-# 10556.0           edge weight 1           additive
-# 140               node count type 1       additive
-# 500               node count type 2       additive
-# 1000              node count type 3       additive
-# 1068              node count type 4       additive
-# 10556             edge count type 1       additive
+# Content of sample meta.json
+
+# Field                   Action
+# ================================
+# binary version          equal
+# node_count              additive
+# edge_count              additive
+# node_type_count         equal
+# edge_type_count         equal
+# node_xxx_feature_count  equal
+# edge_xxx_feature_count  equal
+# partitions count        equal
+# Partition id            equal
+# node weight 1           additive
```

Comment:
why do you keep a list of particular weights, node weight 1, node weight 2, node weight 3 .... ?

Reply:
This isnt really a part of this PR

Reply:
It definitely has to be: this comment explains original meta file contents(line by line, hence the node_weight_1, node_weight_2 comments) and how do they merge into a single one. You are changing meta file format and need to update this comment as well.